### ISSUE TITLE
Replace CKEditor 4 

### ISSUE THREAD

**Issue by bferguso** [archesproject/arches#10828]
CKEditor 4 is end of life as of June 2023: https://ckeditor.com/ckeditor-4/. We should either upgrade to CKEditor 5 of look at alternatives.

**Comment by apeters** [comment]
The latest ckeditor (v 4.23 and greater) now requires a license.  This is currently throwing an error in current implementations of Arches.  [They offer a free license to open source software projects](https://ckeditor.com/wysiwyg-editor-open-source/#:~:text=We%20grant%20a%20free%20CKEditor,needs%20to%20be%20publicly%20available.).  We should either get a license from them or find and alternative to ckeditor.

**Comment by apeters** [comment]
@chiatt and @chrabyrd ☝️ I notice that we just updated the version of ckeditor in package.json

**Comment by chrabyrd** [comment]
> @chiatt and @chrabyrd ☝️ I notice that we just updated the version of ckeditor in package.json

@apeters Yep! It was part of the move to npm, and it was updated to the most recent version that doesn't require a license. This was to resolve a few security issues the previous version was throwing when `npm audit` was run.

I do not have a strong opinion about either obtaining a license from the CKEditor org or moving to a different package, though if we choose to move to a different package [Quill](https://github.com/slab/quill) looks like a good candidate.

**Comment by chrabyrd** [comment]
Doing some more thinking on this and I believe we can probably get away with keeping CKEditor4 past EOL, with the argument that essentially our KOJS components have entered maintenance-only mode. But we should definitely find a modern replacement for integrating into Vue

**Comment by apeters** [comment]
> @apeters Yep! It was part of the move to npm, and it was updated to the most recent version that doesn't require a license.

@chrabyrd it does seem to require  a license though, hence the error message I saw in the console.

**Comment by chrabyrd** [comment]
@apeters  ahh you're right, it's been reverted in #11040
